src/main/java/org/kizombadev/markdownparser/entities/Token.java

```diff
@@ -64,11 +69,15 @@ public String getValue() {
         return value;
     }
 
-    public Category getCategory() {
+    public boolean isTextToken() {
+        return category == Category.TEXT;
+    }
+
+    private Category getCategory() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "getCategory" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)